sample-env

```diff
@@ -0,0 +1,19 @@
+TESTING='True'
```

Comment:
Not necessary against this, but do we really want all of our configurations to go in one file? Probably makes things in the long run to be fair.

Reply:
I don't think we need to address that as of this moment. I think that using a single config file (or environment) should be good enough, since we don't have a multi-hundred line configuration files.

(The fact that we can still fit all of the variables on screen at the same time should say something about the amount of it; looking at you, Amazon).

Reply:
Aha fair enough